Workflow run merge (#1402)
temporalio/documentation: docs/tctl/workflow/run.md

---

> The `tctl workflow run` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and can show Workflow progress.

Reviewer: ```suggestion
The `tctl workflow run` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and can show the progress of a Workflow Execution.
```

The command is entered in the following format:

`tctl workflow run [modifiers]`

**Note:** Start the Worker so the Workflow can make progress.

> (Run `make && ./bin/helloworld -m worker` in samples-go to start the worker)

Reviewer: Let's remove these 2 lines.
Its out of scope and of this content.

> To run a Workflow, the user must specify the following:

Reviewer: ```suggestion
The following are required parameters of the `tctl workflow run` command:
```

1. Task queue name (`--tq`)
2. Workflow type (`--wt`)
3. Execution start to close timeout in seconds (`--et`)

> 4. Input in JSON format (`--i`) (optional)

Reviewer: Make this a list of required parameters only.

```bash

> tctl workflow run --tq hello-world --wt Workflow --et 60 -i '"temporal"'
> 
> # view help messages for workflow run
> gtctl workflow run -h

Reviewer: ```suggestion
tctl workflow run --tq your-task-queue-name --wt YourWorkflowDefinitionName
```

```

Single quotes (`''`) are used to wrap input as JSON.
This command doesn't finish until the Workflow completes.

> ## Modifiers

Reviewer: I actually want to see this header removed from all of these pages.
We don't need a Modifiers grouping header to list all of the modifiers.
